tools/wasme/pkg/util/exec.go

```diff
@@ -11,15 +12,22 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+func container() string {
+	if os.Getenv("WASME_USE_PODMAN") != "" {
+		return "podman"
+	}
+	return "docker"
+}
```

Comment:
Should add a changelog noting that this also implements https://github.com/solo-io/wasm/issues/165 as well as the rust support.